src/utils/filmSort.js

```diff
@@ -0,0 +1,5 @@
+export const filterFilms = (films, filters) => {
+  const isSuitableFilm = (film) => !filters.some(({ type, value }) => (film[type] !== value));
```

Comment:
bad file naming - function inside does not what you expect by filename
I also don't really like the variable naming - better put the function right inside the filter

Reply:
done